lib/mock_redis/version.rb

```diff
@@ -2,5 +2,5 @@
 
 # Defines the gem version.
 class MockRedis
-  VERSION = '0.19.0'.freeze
+  VERSION = '0.20.0'.freeze
```

Comment:
Please don't bump the gem version. We'll do that separately.

Reply:
Got it.